<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Animation</class>
 <widget class="QDialog" name="Animation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting Animation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Begin Step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>End Step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Delay time(ms)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="beginLine"/>
       </item>
       <item>
        <widget class="QLineEdit" name="endLine"/>
       </item>
       <item>
        <widget class="QLineEdit" name="delayTime"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="radioLockView">
     <property name="text">
      <string>Lock Viewport</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioUnlockView">
     <property name="text">
      <string>Unlock Viewport</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoChangeColorCheckBox">
     <property name="text">
      <string>Auto Change Color</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK/Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
